scene/resources/dynamic_font.h

```diff
@@ -54,8 +54,8 @@ class DynamicFontData : public Resource {
 			struct {
 				uint32_t size : 16;
 				uint32_t outline_size : 8;
-				bool mipmaps : 1;
-				bool filter : 1;
+				uint32_t mipmaps : 1;
+				uint32_t filter : 1;
```

Comment:
Can you also pad the struct to a full 32bits? This way we avoid potentially having some uninitialized bits.

```uint32_t unused: 6;```

should do it

Reply:
Sure thing!